### DISCUSSION TITLE
Do you accept non-coreutils commands like `cal`?

### DISCUSSION THREAD

**Discussion by rynewang** [uutils/coreutils#6669] [Ideas]
Hi I found `cal`[1] is not in uutils/coreutils. This makes sense because it's not part of GNU coreutils. But I wonder if you accept a command like that, if I am going to contribute one.

https://www.commandlinux.com/man-page/man1/cal.1.html

**Comment by sylvestre** [comment]
Hello
Yes but under a new project under the uutils umbrella: https://github.com/uutils/
